### ISSUE TITLE
sometimes it fails

### ISSUE THREAD

**Issue by 532910** [Toxblh/youtube-speed-control#9]
Looks like it was runned twice as it shows two values one above another:

![image](https://user-images.githubusercontent.com/14617699/85344174-96e91180-b4f7-11ea-839c-3bfa27a830e9.png)
![image](https://user-images.githubusercontent.com/14617699/85344214-aff1c280-b4f7-11ea-9054-bcacbb57c396.png)

I don't know how to reproduce this.

**Comment by Toxblh** [comment]
Yeah I saw that. Will try to resolve in next release. I not full understand, looks like bug in firefox or JS engine. That is show only one value. But I saw that when I open developer console. And yes, firefox works very bad and sow with opened developer console.

**Comment by Toxblh** [comment]
I catch it, and that is youtube internal "reload" page, in the time also reset a rate and quality of video... :facepalm:
